packages/shared/forks/ReactFeatureFlags.www.js

```diff
@@ -35,6 +35,7 @@ export const {
   favorSafetyOverHydrationPerf,
   disableDefaultPropsExceptForClasses,
   enableNoCloningMemoCache,
+  enableAddPropertiesFastPath,
```

Comment:
did you mean to do this? don’t think this code path is used on www

Reply:
Not really. I just wanted to make it symmetric to `ReactFeatureFlags.native-fb-dynamic.js` - `ReactFeatureFlags.native-fb.js`. I'm not sure if it's actually necessary.